_helpers/dev/cli.mts

```diff
@@ -43,6 +56,32 @@ const test = program.command('test')
       'args to pass to test runner (e.g. --passthru=\'--testNamePattern="testName()"\')'
     )
   )
+  .hook('preAction', (thisCommand) =>{
+    const peprExcellentExamplesRepo = findUpSync('pepr-excellent-examples', {type: 'directory'})
+    execSync('npm install', {cwd: peprExcellentExamplesRepo})
+    if(!process.env.CI)
+    {
+      if(thisCommand.opts().customPackage){
+        const peprBuild = thisCommand.opts().customPackage
+        const peprContainerImage = 'pepr:dev'
+
+        console.log(`Pepr Build under test: ${execSync(`shasum ${peprBuild}`).toString()}\n` +
+        `Pepr Image under test: ${execSync(`docker inspect --format=\'{{.Id}}\' ${peprContainerImage}`).toString()}`)
+        process.env.CUSTOM_PACKAGE = `${peprBuild}`
+      }
+      if(thisCommand.opts().localPackage){
+        const peprRepoLocation = findUpSync('pepr', {type: 'directory'})
+        const peprBuild = 'pepr-0.0.0-development.tgz'
+        const peprContainerImage = 'pepr:dev'
+
+        execSync('npm run build > /dev/null 2>&1', {cwd: peprRepoLocation})
+        execSync(`cp ${peprRepoLocation}/${peprBuild} ${peprExcellentExamplesRepo}`)
+        console.log(`Pepr Build under test: ${execSync(`shasum ${peprExcellentExamplesRepo}/${peprBuild}`).toString()}\n` +
+        `Pepr Image under test: ${execSync(`docker inspect --format=\'{{.Id}}\' ${peprContainerImage}`).toString()}`)
+        process.env.LOCAL_PACKAGE = `${peprExcellentExamplesRepo}/${peprBuild}`
```

Comment:
assign to just one envar? The only difference is if we build or use a provided file.